src/codeFly/fileSystem/Repository.java

```diff
@@ -0,0 +1,191 @@
+package filesystem;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.rmi.activation.UnknownObjectException;
+import java.security.KeyStore.Entry;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.swing.text.AbstractDocument.BranchElement;
+import javax.xml.soap.DetailEntry;
+
+public class Repository {
+    private Map<String, String> loginInfo;
+    private int latestQuestionNum;
+    
+    private static final String rootDirectory = "../Repository/";
```

Comment:
It would be better if the repository is created in the current directory. (i.e. `./Repository`)